public/help/help.md

```diff
@@ -30,3 +31,65 @@ The following procedure describes adding and updating settings of custom tracks.
 4. If simplifying the BED file chunk or graph is possible, users will see a "Simplify Off" button, which when clicked with toggle to "Simplify On". This option enables vg simplify, which would remove small snarls. This option will only appear when there aren't any reads to be displayed. 
 5. Click Go to see the selected tracks render in the visualization area.
 ![Go Button](helpGuideImages/img9.png)
+
+##### How to make link-to-view URLs
+Users can compose URLs that link to a specific view. To do so, users will require specific information about files and tracks. The link has a [query string](https://en.wikipedia.org/wiki/Query_string#Structure) that consists of key-value pairs separated by ampersands, where the keys are not URL-econded and the values are URL-encoded. Keys can use brackets to encode hierarchical structures such as arrays and objects. For arrays, put a number in brackets to assign a new value to or access a value in that entry in the array, and for objects, put the key's name in brackets to assign a new value to or access a value in that entry in the object. The key's name should **not** be in quotes.
+
+These are the fields that can be included in the URL:
+1. `tracks` 
+   Information about tracks. Tracks are objects consisting of trackFile, trackType, and trackColorSettings. To retrieve this information, index the tracks array to access the object and respective keys.
+
+   A track JSON object might look like this:
+   ```
+   {
+      "trackFile": "exampleData/internal/snp1kg-BRCA1.vg.xg",
+      "trackType": "graph",
+      "trackColorSettings": {
+         "mainPalette": "greys",
+         "auxPalette": "ygreys",
+         "colorReadsByMappingQuality": false
+      }
+   }
+   ```
+
+   * The `trackFile` is the path (from the server working directory) or URL (any HTTP/HTTPS URL) to the track file 
+     Examples:
+      * `tracks[0][trackFile]=exampleData/internal/snp1kg-BRCA1.vg.xg`
+      * `tracks[0][trackFile]=https://public.gi.ucsc.edu/~anovak/graphs/trivial-brca1.vg`
+   * The `trackType` specifies the type of the track, which can be "graph", "haplotype", "read"
+      * Example: `tracks[0][trackType]=graph`
+   * The `trackColorSettings` provides color information and is an optional setting.
+      * The color palettes are `mainPalette` and `auxPalette`. 
+         * The colors for the palettes can be hex codes starting with a `#`, or premade palettes: `greys`, `ygreys`, `blues`, `reds`, `plainColors`, or `lightColors`. 
+         * The palettes are used differently for different track types
```

Comment:
All these sub-bullets for the child fields should probably follow the same field-colon-phrase pattern as the top-level bullets. And we want to make sure all the bullets are punctuated consistently; they probably should all end with a period.

And instead of saying in text that something is optional we can always include it in parentheses so it's easy to see at a glance.


```suggestion
   * `trackColorSettings` (optional): the color information for the track. It has subfields:
      * `mainPalette` and `auxPalette` (optional): The "main" and "auxilliary" color palettes to use, respectively. The values for each palette can be a hex code starting with a `#`, or one of the premade palette names: `greys`, `ygreys`, `blues`, `reds`, `plainColors`, or `lightColors`. 
      The palettes are used differently for different track types:
```